main.cpp

```diff
@@ -78,43 +100,87 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
+ */
+void buildHouse(int squareFeet = 0)
+{
+    ignoreUnused(squareFeet);
+}
 
 /*
  2)
- */
+ */
+int caloriesBurned(int hoursSlept)
+{
+    ignoreUnused(hoursSlept);
+    return {};
+}
 
 /*
  3)
- */
+ */
+void scooped()
+{//an empty function for scooping
+}
 
 /*
  4)
  */
+float returnAFloat(char choice = 'y')
+{
+    ignoreUnused(choice);
+    return {};
+}
 
 /*
  5)
- */
+ */
+char getALetter()
+{
+    return {};
+}

```

Comment:
Please provide some function arguments.
Some of the arguments need to have default values
- [ ] revise